Trading/Mode/daily_trading_mode/daily_trading.py

```diff
@@ -244,6 +246,9 @@ def _check_and_adapt_order_details_if_necessary(self, quantity, price, symbol_ma
             decimal.Decimal(f"{quantity}"), decimal.Decimal(f"{price}"), symbol_market)
 
     async def create_new_orders(self, symbol, final_note, state, **kwargs):
+        data = kwargs.get("data", {})
```

Comment:
I don't really like the name `data`, I would prefer something like `params` but it's ok

Reply:
data is what we put there https://github.com/Drakkar-Software/OctoBot-Trading/blob/master/octobot_trading/modes/channel/abstract_mode_consumer.py#L39

Reply:
Right :+1: